src/opcuamodel.cpp

```diff
@@ -763,10 +774,39 @@ void OpcUaModel::browseEnumStrings(QOpcUaNode *node)
                                                             entries[value] = name;
                                                     }
                                                 }
-
-                                                if (!entries.isEmpty())
-                                                    mEnumStringsList[parentNodeId] = entries;
+                                            } else {
+                                                for (const auto &entry : data) {
+                                                    const auto ext =
+                                                            entry.value<QOpcUaExtensionObject>();
+                                                    if (ext.encoding()
+                                                                != QOpcUaExtensionObject::Encoding::
+                                                                        ByteString
+                                                        || ext.encodingTypeId()
+                                                                != QOpcUa::namespace0Id(
+                                                                        QOpcUa::NodeIds::Namespace0::
+                                                                                EnumValueType_Encoding_DefaultBinary))
+                                                        continue;
+
+                                                    auto data = ext.encodedBody();
```

Comment:
Besser anderen Namen wählen, da die data schon in der foreach-Schleife verwendet wird.